server/routes/api/chart.py

```diff
@@ -186,16 +185,47 @@ def get_dates_for_stat_vars(chart_config, sv_data):
     return sv_dates
 
 
-@bp.route('/config/<path:dcid>')
+def build_config(raw_config):
+    """Builds hierachical config based on raw config."""
+    category_map = {}
+    for config in raw_config:
+        is_overview = ('isOverview' in config and config['isOverview'])
+        if is_overview:
+            del config['isOverview']
```

Comment:
nitpick: if config['isOverview'] = False, then this isn't deleted.

Reply:
maybe it'll be cleaner to just build new config objects

Reply:
fixed to check if "isOverview" exists. Build a new object from empty would require knowing all the fields (and new fields if added). This way might be lightweight for change.